### PR TITLE
(Content) Coalition Games News

### PR DESCRIPTION
**Pull request by Arachi-Lover**
----------------------
**Content (News)**

## Summary
Adds some new News entries to the Coalition, related to the Coalition Games, a massive yearly Olympics-like event taking place in July.

### CODE REVIEW

File: data/coalition/coalition news.txt [hunk 1]
```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
...
+	to show
+		month == 7
+	location
+		government "Coalition" "Heliarch"
+	name
+		word
+			"Coalition Games broadcast"
+	message
+		word
+			"A Saryd game is playing on a monitor nearby. The athletes, separated into two teams, gallop around an uneven field, dotted with small mounds and holes, as they try to whack some balls into nets located around the arena edges. Your best guess is that it's an awfully convoluted mix of cricket, football and pool."
```
Comment (tibetiroka) [inline data/coalition/coalition news.txt:355]:
```suggestion
			"A Saryd game is playing on a monitor nearby. The athletes, separated into two teams, gallop around an uneven field, dotted with small mounds and holes, as they try to whack some balls into nets located around the arena's edges. Your best guess is that it's an awfully convoluted mix of cricket, football and pool."
```
I _think_ this is the correct one.

File: data/coalition/coalition news.txt [hunk 2]
```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
...
+			"A crowd has gathered for the transmission, waiting for some kind of race or marathon to start. The camera pans by the competitors, who are all Kimek, then pulls back as the race begins. You judge it to be a pretty standard race initially, until the camera spins around, and it's made clear that the Kimek are running upside down."
+			"Some Arachi attentively wait for a game to start. The screen shows the competitors, who are also Arachi, putting on various layers of plate armor. They then all move to a metal cable, just thick enough for them to grip it with all legs. They then stay there. For many minutes. Until they start falling one by one. Exhilarating."
```
Comment (tibetiroka) [inline data/coalition/coalition news.txt:357]:
```suggestion
			"Some Arachi attentively wait for a game to start. The screen shows the competitors, who are also Arachi, putting on various layers of plate armor. They then all move to a metal cable, just thick enough for them to be gripped with all legs. They then stay there. For many minutes. Until they start falling one by one. Exhilarating."
```
Sounds better imo, but it was fine in terms of grammar.

File: data/coalition/coalition news.txt [hunk 3]
```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
...
+		month == 7
+	location
+		planet "Far Home"
+	name
+		word
+			"Heliarch agent"
+	message
+		word
+			"Dozens of Heliarch agents are guarding the spaceport entrances, occasionally blocking passage as they try to control the flow of people to and from the Coalition Games events."
+			"You spot a few agents on their lunch break inside a local restaurant. They seem to be enjoying Coalition Games broadcast more than the meals themselves."
```
Comment (tibetiroka) [inline data/coalition/coalition news.txt:395]:
```suggestion
			"You spot a few agents on their lunch break inside a local restaurant. They seem to be enjoying the Coalition Games broadcast more than the meals themselves."
```

File: data/coalition/coalition news.txt [hunk 4]
```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
...
+		month == 7
+	location
+		planet "Far Home"
+	name
+		word
+			"Coalition Games crowds"
+	message
+		word
+			`The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs, being helped back to their feet one by one by Heliarch agents and others from the crowd.`
+			`You arrive to a particularly crowded moment of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Batallions of Heliarch agents work as they can to bring everyone back into orderly lines.`
```
Comment (tibetiroka) [inline data/coalition/coalition news.txt:432]:
```suggestion
			`You arrive to a particularly crowded moment of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as they can to bring everyone back into orderly lines.`
```
Also, maybe `hard as they can` or some variation of that might sound better there.

**Review by tibetiroka — CHANGES_REQUESTED** [review]

File: data/coalition/coalition news.txt [hunk 5]
```diff
@@ -428,6 +429,7 @@ news "coalition games crowd"
...
+			"The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs, being helped back to their feet one by one by Heliarch agents and others from the crowd."
+			"You arrive at a particularly crowded part of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as they can to bring everyone back into orderly lines."
+			"Entering the tremendously crowded spaceport, you notice a Saryd pastry shop opening up for business, and decided to head there for a snack. When you finally get to the door, however, they're just done closing. It seems the masses heading to the Coalition Games bought all their products as the line was moving."
```
Comment (tibetiroka) [inline data/coalition/coalition news.txt:434]:
```suggestion
			"Entering the tremendously crowded spaceport, you notice a Saryd pastry shop opening up for business, and decide to head there for a snack. When you finally get to the door, however, they're just done closing. It seems the masses heading to the Coalition Games bought all their products as the line was moving."
```
This is quite different from other news in its style; most news are about First Last observing or about people talking to them, while in this one First Last actually does something.

Also, I'm not sure if `notice` and `decide` should be present or past, but they should probably be in the same tense.

File: data/coalition/coalition news.txt [hunk 6]
```diff
@@ -428,6 +429,7 @@ news "coalition games crowd"
...
+			"Several children of visitors coming to see the Coalition Games are running and playing simplified versions of the events, sporting different colors and assets for different games. Their parents struggle between trying to get them to settle down, and keeping their spots in the waiting lines."
```
Comment (tibetiroka) [inline data/coalition/coalition news.txt:435]:
```suggestion
			"Several children coming to see the Coalition Games are running and playing simplified versions of the events, sporting different colors and assets for different games. Their parents struggle between trying to get them to settle down, and keeping their spots in the waiting lines."
```
`Children of visitors` sound like the children aren't coming to see the games.

**Review by tibetiroka — CHANGES_REQUESTED** [review]

File: data/coalition/coalition news.txt [hunk 7]
```diff
@@ -341,3 +341,95 @@ news "heliarch expeditions past"
...
+		month == 7
+	location
+		planet "Far Home"
+	name
+		word
+			"Coalition Games crowds"
+	message
+		word
+			"The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs, being helped back to their feet one by one by Heliarch agents and others from the crowd."
+			"You arrive at a particularly crowded part of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as they can to bring everyone back into orderly lines."
```
Comment (roadrunner56) [inline data/coalition/coalition news.txt:433]:
```suggestion
			"You arrive at a particularly crowded part of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as hard as they can to bring everyone back into orderly lines."
```

File: data/coalition/coalition news.txt [hunk 8]
```diff
@@ -341,3 +341,95 @@ news "heliarch expeditions past"
...
+			"Entering the tremendously crowded spaceport, you notice a Saryd pastry shop opening up for business, and decided to head there for a snack. When you finally get to the door, however, they're just done closing. It seems the masses heading to the Coalition Games bought all their products as the line was moving."
+			"Several children coming to see the Coalition Games are running and playing simplified versions of the events, sporting different colors and assets for different games. Their parents struggle between trying to get them to settle down, and keeping their spots in the waiting lines."
```
Comment (roadrunner56) [inline data/coalition/coalition news.txt:435]:
```suggestion
			"Several children coming to see the Coalition Games are running and playing simplified versions of the events, sporting different colors and assets for different games. Their parents struggle between trying to get them to settle down and keeping their spots in the waiting lines."
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]
Looks good!

File: data/coalition/coalition news.txt [hunk 9]
```diff
@@ -341,3 +341,95 @@ news "heliarch expeditions past"
...
+		month == 7
+	location
+		planet "Far Home"
+	name
+		word
+			"Coalition Games crowds"
+	message
+		word
+			"The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs, being helped back to their feet one by one by Heliarch agents and others from the crowd."
+			"You arrive at a particularly crowded part of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as hard as they can to bring everyone back into orderly lines."
```
Comment (EjoThims) [inline data/coalition/coalition news.txt:433]:
```suggestion
			"This is a particularly crowded part of the spaceport today. The lines of people waiting for admission into the Games have started crossing and looping in on themselves. Some even head inside shops and restaurants while others stretch into docking areas. Battalions of Heliarch agents work as hard as they can to bring everyone back into orderly lines."
```

File: data/coalition/coalition news.txt [hunk 10]
```diff
@@ -341,3 +341,95 @@ news "heliarch expeditions past"
...
+			"Entering the tremendously crowded spaceport, you notice a Saryd pastry shop opening up for business, and decided to head there for a snack. When you finally get to the door, however, they're just done closing. It seems the masses heading to the Coalition Games bought all their products as the line was moving."
+			"Several children coming to see the Coalition Games are running and playing simplified versions of the events, sporting different colors and assets for different games. Their parents struggle between trying to get them to settle down and keeping their spots in the waiting lines."
```
Comment (EjoThims) [inline data/coalition/coalition news.txt:435]:
```suggestion
			"Several children coming to see the Coalition Games are running and playing simplified versions of the events, sporting different colors and assets for different games. Their parents struggle  to get them to settle down while keeping their spots in the waiting lines."
```

**Review by EjoThims — CHANGES_REQUESTED** [review]

File: data/coalition/coalition news.txt (hunk 10 above)
Comment (tibetiroka) [reply data/coalition/coalition news.txt:435]: Look out for that extra space

**Review by EjoThims — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]

File: data/coalition/coalition news.txt [hunk 11]
```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
...
+	to show
+		month == 7
+	location
+		government "Coalition" "Heliarch"
+	name
+		word
+			"Coalition Games broadcast"
+	message
+		word
+			"A Saryd game is playing on a monitor nearby. The athletes, separated into two teams, gallop around an uneven field, dotted with small mounds and holes, as they try to whack some balls into nets located around the arena's edges. Your best guess is that it's an awfully convoluted mix of cricket, football and pool."
```
Comment (Amazinite) [inline data/coalition/coalition news.txt:355]:
```suggestion
			"A Saryd game is playing on a monitor nearby. The athletes, separated into two teams, gallop around an uneven field, dotted with small mounds and holes, as they try to whack some balls into nets located around the arena's edges. Your best guess is that it's an awfully convoluted mix of cricket, football, and pool."
```

File: data/coalition/coalition news.txt [hunk 12]
```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
...
+			"A crowd has gathered for the transmission, waiting for some kind of race or marathon to start. The camera pans by the competitors, who are all Kimek, then pulls back as the race begins. You judge it to be a pretty standard race initially, until the camera spins around, and it's made clear that the Kimek are running upside down."
+			"Some Arachi attentively wait for a game to start. The screen shows the competitors, who are also Arachi, putting on various layers of plate armor. They then all move to a metal cable, just thick enough for them to be gripped with all legs. They then stay there. For many minutes. Until they start falling one by one. Exhilarating."
+			"During a break from the games, some analysts go over some spreadsheets with the statistics for the competitors of the past few games. You can't read any of it, but passerby workers and captains stay a while to glance at the results."
```
Comment (Amazinite) [inline data/coalition/coalition news.txt:358]:
```suggestion
			"During a break from the games, some analysts go over spreadsheets with the statistics for the competitors of the past few games. You can't read any of it, but passerby workers and captains stay a while to glance at the results."
```

File: data/coalition/coalition news.txt [hunk 13]
```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
...
+			"Local Kimek"
+			"Local Arach"
+	message
+		word
+			`"Going to see the Coalition Games, I am. Our biggest yearly event, it is."`
+			`"Competing in the Coalition Games this year, a friend of mine is. Cheering for them, my whole family is."`
+			`"Seen the latest transmission of the Coalition Games, have you? Throughout it all, on the edge I was!"`
+			`"Checked the job board lately, have you? Eager to see the Coalition Games, tons of passengers are. A good season for business, this is."`
+			`"Watching the Coalition Games transmissions lately, I have been. Tried to become an athlete myself, I once did, but too rigorous, their training routine is! Made me appreciate the sports more, it has."`
+			`"Just arrived from my first live viewing of the Coalition Games, I have. An incredible experience, it was. Go again next year, I for sure will!"`
```
Comment (Amazinite) [inline data/coalition/coalition news.txt:382]:
```suggestion
		word
			`"`
		word
			`Going to see the Coalition Games, I am. Our biggest yearly event, it is.`
			`Competing in the Coalition Games this year, a friend of mine is. Cheering for them, my whole family is.`
			`Seen the latest transmission of the Coalition Games, have you? Throughout it all, on the edge I was!`
			`Checked the job board lately, have you? Eager to see the Coalition Games, tons of passengers are. A good season for business, this is.`
			`Watching the Coalition Games transmissions lately, I have been. Tried to become an athlete myself, I once did, but too rigorous, their training routine is! Made me appreciate the sports more, it has.`
			`Just arrived from my first live viewing of the Coalition Games, I have. An incredible experience, it was. Go again next year, I for sure will!`
		word
			`"`
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Review by Amazinite — APPROVED** [review]

**Pull request merged**